### PR TITLE
Added yaml configuration.

### PR DESCRIPTION
**Pull request by bjlittle**
@pelson 

I suspect one of the integration tests will fail as it will need a `config.yaml` file ... so I'm expecting to push a patch.

### CODE REVIEW

File: conda_rpms/build_rpm_structure.py [hunk 1]
```diff
@@ -8,26 +8,72 @@
...
+from conda_gitenv.resolve import tempdir, create_tracking_branches
+from git import Repo
+import yaml
 
 import generate
 import install as conda_install
 
 
-def create_rpmbuild_for_env(pkgs, target):
+class Config(dict):
```
Comment (pelson) [inline conda_rpms/build_rpm_structure.py:28]: Gold plated config? 😜

Comment (bjlittle) [inline conda_rpms/build_rpm_structure.py:28]: Guilty as charged ... :smile:

**Comment by pelson** [comment]
Looks like the traceback isn't so good, but:

```
error: failed to stat /repo/tests/output/gitenv_rpmbuild/SPECS/*.spec: No such file or directory

Saving Primary metadata
Saving file lists metadata
Saving other metadata
Generating sqlite DBs
Sqlite DBs complete
Loaded plugins: fastestmirror, ovl
Setting up Install Process
Determining fastest mirrors
No package SciTools-env-* available.
Error: Nothing to do

./tests/gitenv_build_rpms.sh returned exit code 1
```

I'm really happy with ee9afc23b493167d1157a3a722154466547956e1, and ask that you add new commits rather than modify that one. 👍

**Comment by pelson** [comment]
https://github.com/bjlittle/conda-rpms/blob/064daba5eccdb06ec77d1ec3d20a88c0f156911b/tests/gitenv_build_rpms.sh#L32 😉

**Comment by bjlittle** [comment]
@pelson Yup, thanks, just found it and changing it :wink:

**Comment by bjlittle** [comment]
@pelson Are you happy with adding the `config.yaml.example` file?

I didn't quite know the approach to take in order to give readers a template of a `config.yaml` file that we expect ...

**Comment by bjlittle** [comment]
@pelson Thar she blows :whale: ... looks good to me.

**Comment by pelson** [comment]
Sure is 👍

**Pull request merged**